.github/workflows/release.yaml

```diff
@@ -11,9 +15,25 @@ jobs:
       uses: actions/setup-go@v4
       with:
         go-version: "1.20"
-
     - uses: actions/checkout@v3
 
+    - name: Generate bundle
+      run: make bundle
+      env:
+        OPERATOR_VERSION: ${{inputs.version}}
+        IMG_PREFIX: ghcr.io/grafana/tempo-operator
+
+    - name: Generate CHANGELOG
+      run: make chlog-update
+      env:
+        OPERATOR_VERSION: ${{inputs.version}}
+
+    # Commit all changed files back to the repository
+    - uses: stefanzweifel/git-auto-commit-action@v4
```

Comment:
On a second thought, could we replace this with plain `git` commands? In terms of securing our build pipeline, I think it's best to avoid any third-party actions/components. Maybe at some point this external repo gets compromised and then we would ship malicious code in all future releases.

afaics this action just does the following (untested):
```
git commit -m "Release ${{inputs.version}}"
git push
git tag v${{inputs.version}}
git push v${{inputs.version}}
```

Reply:
Changed

Reply:
Perfect, thanks! :100: